client/src/components/ColorList.js

```diff
@@ -76,6 +107,32 @@ const ColorList = ({ colors, updateColors }) => {
           </div>
         </form>
       )}
+
+      {!editing && (<form onSubmit={addColor}>
+          <legend>add color</legend>
+          <label>
+            color name:
+            <input
+              onChange={e =>
+                setColorToAdd({ ...colorToAdd, color: e.target.value })
+              }
+              value={colorToAdd.color}
+            />
+          </label>
+          <label>
+            hex code:
+            <input
+              onChange={e =>
+                setColorToAdd({
+                  ...colorToAdd,
+                  code: { hex: e.target.value }
+                })
+              }
+              value={colorToAdd.code.hex}
+            />
+          </label>
+          <button className='add-color-btn'>Add Color</button>
+        </form>)}
```

Comment:
having a div somewhere in here that was a small square with a background color set to the hex value of input state is what I'm talking about.